### DISCUSSION TITLE
Welcome to Website Discussions!

### DISCUSSION THREAD

**Discussion by theovincent** [SamuelDiai/Dash-Website#7] [General]

**Comment by yiminzhao** [comment]
Could you share the summary-level GWAS results of the nine multidimensionality of Aging (https://www.multidimensionality-of-aging.net/)?
The GWAS results of abdomen age have been made available on Figshare.
Thanks!

**Comment by excellent-song** [comment]
Thank you very much for your contributions! I would like to know the detailed pre-processing procedure for each type of image data, especially abdominal data and cardiac data. I'd love to know how you handle it in the code!
Thank you very much!
